packages/fxa-settings/src/pages/Signin/SigninUnblock/container.tsx

```diff
@@ -91,22 +91,33 @@ const SigninUnblockContainer = ({
     };
 
     // Get credentials with the correct key version
+    const status = await (async () => {
```

Comment:
Just curious, why make this an IIFE instead of `let status;` with `try`/`catch`?

Reply:
Cleaner IMO